### PR TITLE
Added login, sigup page aong with auth context

### PR DESCRIPTION
**Pull request by anupkgurung**

### CODE REVIEW

**Review by Rohitprasad83 — APPROVED** [review]
Code looks clean, good to merge

File: src/pages/Login/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            [id] : value
+        }))
+    }
+    const navigateTo = useNavigate();
+    const handleLogin = async (e) => {
+        e.preventDefault();
+        try{
+            const { data } = await axios.post("/api/auth/login",userCredentials);
+            console.log(data)
+            //localStorage.setItem("token",encodedToken);
```
Comment (Aditi0505) [inline src/pages/Login/Login.jsx:24]: Please remove if not needed.

File: src/pages/Login/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            navigateTo("/");
+        }catch(error){
+            console.log(error);
```
Comment (Aditi0505) [inline src/pages/Login/Login.jsx:27]: Display error on the UI

File: src/pages/Login/Login.jsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                title="Email"
+                type="email"
+                onChange={handleChange}
+                value={userCredentials.email}
+                placeholder="username@email.com"
+                hasAutoComplete={false}
+                isRequired={true}
+                />
+
+            <div className="">
```
Comment (Aditi0505) [inline src/pages/Login/Login.jsx:48]: Remove className if not needed.

File: src/pages/Login/Login.jsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+                    hasAutoComplete={false}
+                    isRequired={true}
+                    hasOnlyInput={true}
+                 />                  
+            </div>
+            <div className="remember-credentials-check flex-row padding-top-2">
+                <input type="checkbox" id="remember" className="checkbox" />
+                <label htmlFor="remember">Remember Me</label>
+            </div>
+            <div className="">
```
Comment (Aditi0505) [inline src/pages/Login/Login.jsx:69]: Same as above

File: src/pages/Signup/Signup.jsx [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+    const { setUser } = useAuthentication();
+    const handleSignup = async (e) => {
+        e.preventDefault();
+        try{
+            const {data : {createdUser ,encodedToken}} = await axios.post("/api/auth/signup",userCredentials);
+            localStorage.setItem("token",encodedToken);
+            setUser(createdUser);
+            navigateTo("/")
+        }catch(error){
+            console.log(error);
```
Comment (Aditi0505) [inline src/pages/Signup/Signup.jsx:29]: Display the error on the UI

File: src/pages/Signup/signup.css [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    margin-top: 3rem;
+    padding: 1rem;
+}
+.login-form , .signup-form {
+    width: 30rem;
+    margin: 5rem;
+    padding: 3rem 3rem;
```
Comment (Aditi0505) [inline src/pages/Signup/signup.css:14]: You can remove the extra 3rem and use shorthand property.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions. rest looks good to me.
Also, add description and screenshot for better visualization.

**Pull request merged**